src/main/kotlin/com/github/erosb/jsonsKema/SchemaVisitor.kt

```diff
@@ -113,10 +113,10 @@ abstract class SchemaVisitor<P> {
     open fun visitUnevaluatedPropertiesSchema(schema: UnevaluatedPropertiesSchema): P? = visitChildren(schema)
     open fun visitFormatSchema(schema: FormatSchema): P? = visitChildren(schema)
 
-    open fun identity(): P? = null
+    open fun identity(parent: Schema): P? = null
```

Comment:
It looks like a little backward-compatibility break. Not sure if it would break anyone's code. WDYT about keeping BC like:
```

    open fun identity(): P? = null
    open fun identity(parent: Schema): P? = identity()
```
?

Reply:
Sounds good, I'll make the change